assets/externalized/strategic-map-creature-lairs.json

```diff
@@ -4,10 +4,10 @@
  * Field definitions:
  *  - lairId:    A unique numeric ID for the lair. Must be an integer greater than zero.
  *  - associatedMineId:    Creature lair must be located and associated with one mine.
- *  - entranceSector:    Where the lair entrance is located. There must be an alternate map file (*_a.dat), which will be activated at lair initialization.
+ *  - entranceSector:    Where the lair entrance is located. There must be an alternate map file (_a.dat), which will be activated at lair initialization.
```

Comment:
Why is this needed? Is `*` invalid in comments / schema?

Reply:
You catched me :grin:. It should be perfectly valid but there is a [bug with the rust json reader](https://github.com/tmccombs/json-comments-rs/pull/9), which I am currently using to build an editor based on the JSON data: https://github.com/selaux/stracciatella-toolset